sklearn/metrics/scorer.py

```diff
@@ -201,8 +201,7 @@ def _passthrough_scorer(estimator, *args, **kwargs):
     return estimator.score(*args, **kwargs)
 
 
-def check_scoring(estimator, scoring=None, allow_none=False,
```

Comment:
If it does not begin with an underscore or is not in ``sklearn.utils`` it should be documented and part of public API. I think all non-underscore functions should be documented.